### DISCUSSION TITLE
What is the allowed latency for raft between nodes?

### DISCUSSION THREAD

**Discussion by gsogol** [rabbitmq/ra#370] [General]
How many ms is allowed in terms of latency between nodes in raft? Many distributed systems who use it say 8 ms max but curious what regarding Rabbit. Thanks

**Comment by kjnilsson** [comment]
There is no general recommendation apart from "lower is better".

I suspect the system would work better if the latency is approximately the same between all nodes.

**Reply by michaelklishin** [reply]
8ms sounds reasonable but it is equally important to know the distribution: if this is a median? a 90th percentile? 95th? 99th?

Large spikes in latency of even one follower can result in severe throughput drops, and potentially even result in unnecessary leader elections, which delays operations on quorum queues.
